dockers/docker-base/Dockerfile.j2

```diff
@@ -28,12 +28,9 @@ RUN apt-get -y install \
     vim-tiny           \
     perl               \
     python             \
+    rsyslog            \
     less
 
-# Install a newer version of rsyslog from jessie-backports in hopes of
-# eliminating memory leaks
```

Comment:
Do we need to update rsyslog version?

Reply:
We may want to consider it for the 201811 branch, because the Docker images are still based on Jessie (I just did this for the 201803 branch).

For the master branch, we are upgrading all images to Stretch, which has the newer version of rsyslog.

Reply:
"I just did this for the 201803 branch"
How can I upgrade syslog now  for the 201811 branch

Reply:
@yinshuangshuang: Thank you for the suggestion! I have created a PR to upgrade the version of rsyslog installed in the Docker containers in the 2018011 branch here: https://github.com/Azure/sonic-buildimage/pull/3127.